src/index.html

```diff
@@ -7,6 +7,42 @@
     <title>Document</title>
 </head>
 <body>
-    <h1>LeaderBoard</h1>
+    <div class="main">
+        <div class="container">
+            <h1>LeaderBoard</h1>
+            <div class="leaderboard">
+                <div class="scores">
+                    <div class="refresh">
+                        <h2>Recent scores</h2>
+                        <button>Refresh</button>
+                    </div>
+                    <div class="list-scores">
+                        <ul>
+                            <li>Name: 100</li>
+                            <li>Name: 70</li>
+                            <li>Name: 90</li>
+                            <li>Name: 50</li>
+                            <li>Name: 85</li>
+                            <li>Name: 77</li>
+                            <li>Name: 98</li>
+                            <li>Name: 89</li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="add-score">
+                    <h2>Add Your Scores</h2>
+                    <div class="form-add">
+                        <form>
+                          <div><input type="text" placeholder="Your name"></div>  
+                          <div><input type="text" placeholder="Your score"></div>  
+                          <div><input type="submit" value="Submit"></div>  
+                        </form>
+                    </div>
+                </div>
+            </div>
+            
+        </div>
+    </div>
+   
```

Comment:
- [ ] On this part, please, add the script to link `index.js` with `index.html`
- [ ] At the same time, is important that all project will be built on js code, here I could observe how all the code is on `index.html`, try to use js to this. This will help for future projects because using js to write HTML files is a better practice.
- Hint: index.html main contain an only root element
- Don't forget to use ES6 best practices when you work on this.